rfcs/text/0352-graph-type-system.md

```diff
@@ -116,12 +109,8 @@ Given the motivation above and some other implications of the current design, th
 
 As well as addressing these existing shortcomings, this RFC seeks to:
 
-<!-- markdownlint-disable  MD029 -->
-
-3.  define a constrained type system that is able to describe any JSON object structure
-4.  refine the current way of defining links between entities, encapsulating the approach within the type system
-
-<!-- markdownlint-enable  MD029 -->
```

Comment:
What prompted this change? These numbers were meant to be a continuation of the numbers above.

Reply:
<img width="1434" alt="Screenshot 2023-04-20 at 12 30 10" src="https://user-images.githubusercontent.com/127305333/233353224-ba4baebb-8664-49db-baa4-3d310f9605c8.png">

Reply:
The problem is the HTML comment lines disabling markdownlint rules. As the error message says, comments in MDX are expected to be in a different syntax.

I can think of two options here:
1. Rename the files to `.mdx` – let's try and avoid that for now as it will probably have wider implications
2. Deal with the `markdownlint` disable rules some other how, so we don't have to include HTML comments in the file. I suggest trying this first as it will hopefully be simpler.

You can disable `markdownlint` rules via config files as well as comments in the doc. We have one such file, `.markdownlint.json`, in the root of the repo. I suggest that we don't disable the rules across the whole repo, though, as they are still useful to have. Let's try adding a new `.markdownlint.json` file inside `rfcs` that just disables the rules we are currently disabling via HTML comments. 

You should be able to extend the root config file, and add the two new disable options – see [docs here](https://github.com/DavidAnson/markdownlint#optionsconfig)

Please let me know if you have any issues.

Reply:
Hello @CiaranMn, the second option has been implemented but it doesn't serve the function, the command `lint:markdownlint` completely disregards the `.markdownlint.json` config file in the `rfcs` folder

Reply:
I've pushed a change [here](https://github.com/blockprotocol/blockprotocol/pull/1255/files/02d4868b9b206d6e85114a8a49ee4f1e0864ca1e..a7a08585c9b5d7765567966ad8b1f7932cb98904) to replace the CLI used to run `markdownlint`. The previous one wasn't checking nested config files, the new one does.

Please restore the content deleted previously and then it should be good to review again.

Reply:
I pushed https://github.com/blockprotocol/blockprotocol/pull/1255/commits/edd7efd81d09c872c956073d08c2def103ec668b to fix a linting issue